packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/List.tsx

```diff
@@ -44,38 +44,32 @@ export const SlicesList: React.FC<SlicesListProps> = ({
   return (
     <Grid
       elems={slices}
-      defineElementKey={(slice: SliceZoneSlice) => {
+      defineElementKey={(slice) =>
+        slice.type === "Slice"
```

Comment:
So we are sure we don't support opening old slice in the builder, correct?

It was like that before but since Lucie is working on that do you think it impacts this line?